### DISCUSSION TITLE
PlatormIO Example Build errors

### DISCUSSION THREAD

**Discussion by zjwhitehead** [earlephilhower/arduino-pico#492] [Q&A]
I'm having trouble using PlatformIO with my Adafruit Feather RP2040. 
Even when running the example here https://github.com/maxgerhardt/pio-pico-earlephilhower-test-all-variants
I get a build error 
`Error: Unknown board ID 'adafruit_kb2040'` - same thing for all boards if I comment that one out.

Whats the trick to getting PIO working? I've tried some of the latest releases. 
Do we still need more changes merged in?

**Comment by maxgerhardt** [comment]
If you have an unknown board ID, it may be because you have an outdated git version of the platform. Remove all `<user directory>/.platformio/platforms/raspberrypi*` folders and retry building.

**Reply by zjwhitehead** [reply]
Thanks! I think that helped. Its now working on my mac but on my main windows machine I get 
`'arm-none-eabi-g++' is not recognized as an internal or external command,
operable program or batch file.`
I may have to nuke me PIO installation and start over here

**Reply by maxgerhardt** [reply]
Your compiler package may be corrupted as well, or you told it to download the wrong package. Remember, the toolchain download links are OS-specific and the linked example has the link for *Windows*. As a first step, you should remove all `<user directory>/.platformio/packages/toolchain-pico*` folders.
